stylesheets/components/avatar.css

```diff
@@ -0,0 +1,63 @@
+@import url("/stylesheets/main.css");
+
+.avatar {
+  margin: 1rem;
+  width: 3rem;
+  height: 3rem;
+  border-radius: 100%;
+  position: relative;
+}
+
+.avatar-sm {
+  width: 2rem;
+  height: 2rem;
+}
+
+.avatar-lg {
+  width: 3.5rem;
+  height: 3.5rem;
+}
+
+.avatar-xl {
+  width: 4rem;
+  height: 4rem;
+}
+
+.avatar-sq {
+  border-radius: 5%;
+}
+
+.avatar-container {
+  position: relative;
+  width: fit-content;
+}
+
+.badge {
+  width: 15px;
+  height: 15px;
```

Comment:
This could be changed to `rem`.

Reply:
Rest all looks fine to me.

Reply:
Converted px into rem as suggested , thanks for reviewing.